README.md

```diff
@@ -66,6 +68,27 @@ $ make BUILD_AVX=1 check
         [ Verified ]
 ```
 
+* Test with Arrow SoCKit board with [Download](https://github.com/LampaLab/iota_fpga/releases/tag/v0.1) Linux sd-card image, root password is 123456 and you need to download dcurl into root directory. 
```

Comment:
Should we update RBF? => https://github.com/LampaLab/iota_fpga/releases/tag/v0.3

Reply:
Yes, we need. Where and how can we upload this file?

Reply:
Yes, we need. Where and how can we upload this file?

Reply:
I have used the performance tool [0] supported by Lampa Lab in SoCkit board with the synthesized RBF (v0.3) and reproduced experimental data depicted to a figure, e.g. [1].
[0] https://github.com/LampaLab/iota_fpga/tree/master/pow_accel_soc/software/pow_performance_test
[1] https://github.com/LampaLab/iota_fpga/blob/master/pow_accel_soc/software/pow_performance_test/hist_14mwm.png

Reply:
You'd better create a new Markdown file (named after `FPGA-ACCEL.md`) as placeholder of Lampa Lab stuff and mention it within `README.md`.

Reply:
I need to write what content is written in `FPGA-ACCEL.md` and where it is placed.

Reply:
Ditto.

Reply:
ping? Any further changes reflecting the above?